develop/schemadoc/Protocol/Protocol.Triggers.Trigger.Time.md

```diff
@@ -64,9 +64,9 @@ Not all Protocol.Triggers.Trigger.Time values can be used in combination with th
 
 In the code above, the Time element is the hexadecimal representation of the following string:
 
-Error : 12029. [ERROR_WINHTTP_CANNOT_CONNECT]
+`Error : 12029. [ERROR_WINHTTP_CANNOT_CONNECT]`
 
 > [!NOTE]
 >
 > - If you define a trigger that will be activated by another trigger, then leave this tag empty.
-> - Prior to DataMiner 9.0.0 [CU2], when `<NoTimeout>...</NoTimeOut>` is specified in the protocol, the "timeout after retries" trigger will not go off (whereas the "timeout" trigger will go off) when the response matches the value set in `<NoTimeout>`. However, from DataMiner 9.0.0 [CU2] (RN 12543) onwards, the "timeout after retries" trigger will go off despite the `<NoTimeout>` setting, resulting in similar behavior as the "timeout" trigger.
+> - In legacy DataMiner versions prior to DataMiner 9.0.0 [CU2], when `<NoTimeout>...</NoTimeOut>` was specified in the protocol, the "timeout after retries" trigger did not go off (whereas the "timeout" trigger did go off) when the response matched the value set in `<NoTimeout>`. However, in the currently supported DataMiner versions, the "timeout after retries" trigger will go off despite the `<NoTimeout>` setting, resulting in similar behavior as the "timeout" trigger.<!-- RN 12543 -->
```

Comment:
@PedroDebevere, I've rephrased this to be able to keep this info in case it's still relevant; however, it might be better to just remove this entirely. What do you think?